OptimizelySDK/OptimizelyTests/OptimizelyTests-APIs/OptimizelyClientTests_Others.swift

```diff
@@ -229,13 +229,9 @@ class OptimizelyClientTests_Others: XCTestCase {
         let exp = expectation(description: "a")
         
         optimizely.start { result in
-            switch result {
-            case .success:
-                XCTAssert(false)
-            case .failure(OptimizelyError.dataFileVersionInvalid):
+            
+            if case .failure(OptimizelyError.dataFileVersionInvalid) = result  {
```

Comment:
With this change, we cannot catch error on .success case

Reply:
Changes made.